### ISSUE TITLE
White text for cancelled order status on account page 

### ISSUE THREAD

**Issue by curtiswilkinson** [NetoECommerce/Skeletal#95]
On template: 
Skeletal/private/www/netosuite/Templates/skeletal/customer/template.html

there is the following:
```
 <span class="label pull-right
<!--##[%DATA id:'order_status' if:'eq' value:'On Hold'%]##-->label-warning<!--##[%END DATA%]##-->
<!--##[%DATA id:'order_status' if:'eq' value:'Pick'%]##-->label-info<!--##[%END DATA%]##-->
<!--##[%DATA id:'order_status' if:'eq' value:'New'%]##-->label-primary<!--##[%END DATA%]##-->
<!--##[%DATA id:'order_status' if:'eq' value:'Backorder Approved'%]##-->label-warning<!--##[%END DATA%]##-->
 <!--##[%DATA id:'order_status' if:'eq' value:'New Backorder'%]##-->label-warning<!--##[%END DATA%]##-->
<!--##[%DATA id:'order_status' if:'eq' value:'Pack'%]##-->label-info<!--##[%END DATA%]##-->
<!--##[%DATA id:'order_status' if:'eq' value:'Dispatched'%]##-->label-success<!--##[%END DATA%]##-->
">[@order_status@]<!--##[%DATA id:'refund_status' if:'ne' value:''%]##--> ([@refund_status@])<!--##[%END DATA%]##--></span>
```

This doesn't have a "cancelled" order status and causes the following:
![githubcancelled](https://cloud.githubusercontent.com/assets/12440573/11620283/1c7e7104-9cfd-11e5-9460-1ee2cfd230d6.png)

 As I see it, these are the 2 easiest options

1 Give Cancelled a labal-info or warning class

```
 <span class="label pull-right
<!--##[%DATA id:'order_status' if:'eq' value:'On Hold'%]##-->label-warning<!--##[%END DATA%]##-->
<!--##[%DATA id:'order_status' if:'eq' value:'Pick'%]##-->label-info<!--##[%END DATA%]##-->
<!--##[%DATA id:'order_status' if:'eq' value:'New'%]##-->label-primary<!--##[%END DATA%]##-->
<!--##[%DATA id:'order_status' if:'eq' value:'Backorder Approved'%]##-->label-warning<!--##[%END DATA%]##-->
 <!--##[%DATA id:'order_status' if:'eq' value:'New Backorder'%]##-->label-warning<!--##[%END DATA%]##-->
<!--##[%DATA id:'order_status' if:'eq' value:'Pack'%]##-->label-info<!--##[%END DATA%]##-->
<!--##[%DATA id:'order_status' if:'eq' value:'Dispatched'%]##-->label-success<!--##[%END DATA%]##-->
<!--##[%DATA id:'order_status' if:'eq' value:'Cancelled'%]##-->label-info<!--##[%END DATA%]##-->
">[@order_status@]<!--##[%DATA id:'refund_status' if:'ne' value:''%]##--> ([@refund_status@])<!--##[%END DATA%]##--></span>
```

2) Make the normal "label" class dependant on these if statements so the cancelled order status appears in black text

```
 <span class=" pull-right
<!--##[%DATA id:'order_status' if:'eq' value:'On Hold'%]##-->label label-warning<!--##[%END DATA%]##-->
<!--##[%DATA id:'order_status' if:'eq' value:'Pick'%]##-->label label-info<!--##[%END DATA%]##-->
<!--##[%DATA id:'order_status' if:'eq' value:'New'%]##-->label label-primary<!--##[%END DATA%]##-->
<!--##[%DATA id:'order_status' if:'eq' value:'Backorder Approved'%]##-->label label-warning<!--##[%END DATA%]##-->
 <!--##[%DATA id:'order_status' if:'eq' value:'New Backorder'%]##-->label label-warning<!--##[%END DATA%]##-->
<!--##[%DATA id:'order_status' if:'eq' value:'Pack'%]##-->label label-info<!--##[%END DATA%]##-->
<!--##[%DATA id:'order_status' if:'eq' value:'Dispatched'%]##-->label label-success<!--##[%END DATA%]##-->
">[@order_status@]<!--##[%DATA id:'refund_status' if:'ne' value:''%]##--> ([@refund_status@])<!--##[%END DATA%]##--></span>
```

**Comment by brandonsheppard** [comment]
@ryvanvel Assigning this to you—you should update the logic to the new syntax while you're there :wink:

**Comment by ryvanvel** [comment]
Fixed & comitted with f100a7b
